public/js/monitor.js

```diff
@@ -561,8 +556,8 @@ function getPageAttribution () {
         e.preventDefault()
 
         // Send GA Ping
-        if (typeof (ga) !== 'undefined') {
-          ga('send', {
+        if (typeof (window.ga) !== 'undefined') {
```

Comment:
nit: I wonder if we should expose some helper for this `typeof (window.ga) !== 'undefined'` since it looks like we're using it in 8-9 places in this PR.

Reply:
Indeed, I plan to address this in an upcoming ticket for complete analytics refactor – since GA is being deprecated next year anyway.